docker/docker-compose.prod.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.8'
+
```

Comment:
Try adding `restart: on-failure` for cases when it fails

Reply:
> Try adding `restart: on-failure` for cases when it fails

I've added the restart property for production, before I've added it for development too but now I've removed it

Reply:
> Try adding `restart: on-failure` for cases when it fails

I've added the restart property for production, before I've added it for development too but now I've removed it